examples/tutorials/a_atoms.py

```diff
@@ -0,0 +1,32 @@
+import autode as ade
+
+# Create a carbon atom
+atom = ade.Atom('C')
+
+# autodE atoms have a position
+print('Position:  ', atom.coord)
+
+# and useful properties, like the atomic number. See all of them
+# here: https://duartegroup.github.io/autodE/reference/atoms.html
+print('Z:         ', atom.atomic_number)
+
+# and can be translated by a 3D vector
```

Comment:
Does this translate _by_ (1, 0, 0) or _to_ (1, 0, 0)? I'm guessing by but perhaps the comment could be more explicit about this